lib/less/utils.js

```diff
@@ -36,6 +38,28 @@ var utils = {
         }
         return cloned;
     },
+    copyOptions: function(obj1, obj2) {
+        var opts = utils.defaults(obj1, obj2);
+        if (opts.strictMath) {
+            opts.math = MATH.STRICT_LEGACY;
+        }
+        if (opts.hasOwnProperty('math') && typeof opts.math === 'string') {
+            switch (opts.math.toLowerCase()) {
+                case 'always':
+                    opts.math = MATH.ALWAYS;
+                    break;
+                case 'parens-division':
+                    opts.math = MATH.PARENS_DIVISION;
+                    break;
+                case 'parens-all':
```

Comment:
We had talked about `strict` as an alias of `parens`/`parens-all`.  I don't think it's totally necessary; I'm just making sure it was a conscious choice to leave it out.

Reply:
It was, but after sleeping on it, I realized it might make the concept easier for people switching to decide which one to use.